autoload/ollama/edit.vim

```diff
@@ -5,6 +5,31 @@ let s:buf = -1
 " avoid starting a 2nd edit job while one is in progress
 let g:edit_in_progress = 0
 
+if !exists('g:ollama_embedded_python') || g:ollama_embedded_python == 0
```

Comment:
Perhaps it would be better to perform the "is embedded Python" available check as the first expression of the corresponding function?

This way it would be closer to the reset of the relevant code.
When a function that uses embedded Python is added or removed, it is easier to see that one also needs to think about a case when Python is not available.